nasa-gibs/worldview
# WV-2944: Updating to OL v8.2.0 & remove NASA Logo for static maps in EIC Mode

## Description

OL v8 is currently breaking our logic for displaying a static image layer of Blue Marble for EIC mode. This will update to OL v8.2.0 where this feature now works.

This also conditionally removes the NASA logo when we are displaying the static map in EIC mode. 

## How To Test

1. `git checkout wv-2944`
2. `npm ci`
3. `npm run watch`
4. I have configured the `CERES_EBAF_TOA_Shortwave_Flux_All_Sky_Monthly` layer for static map testing in EIC mode since it has not had coverage for several years. Use this to load EIC mode with this layer. http://localhost:3000/?v=-181.74360912131363,-98.53068072538338,182.80846643543055,106.52986177528524&df=true&kiosk=true&eic=si&l=CERES_EBAF_TOA_Shortwave_Flux_All_Sky_Monthly&lg=false
5. Give WV 3 to 5 seconds to cycle the API calls and then verify that the Blue Marble static map loads without the WV logo.

@nasa-gibs/worldview
